### PR TITLE
Update PPMiniLM performance result tested by QA

### PR DESCRIPTION
**Pull request by LiuChiachi**
### PR types

Others
### PR changes

Docs
### Description

1.Update PPMiniLM performance result tested by QA
2.fix default params in serving export

### CODE REVIEW

File: examples/model_compression/pp-minilm/README.md [hunk 1]
```diff
@@ -60,7 +60,9 @@ PP-MiniLM 压缩方案以面向预训练模型的任务无关知识蒸馏(Task-a
 
 2.量化后比量化前模型参数量多了 0.1M 是因为保存了 scale 值；
 
-3.性能测试的条件是：batch_size: 32, max_seq_len: 128。
+3.性能测试的条件是：batch_size: 32, max_seq_len: 128；
+
+4.PPMiniLM 模型均接入了FasterTokenizer，FasterTokenizer 对模型的精度没有影响，但是会加快推理速度。
```
Comment (ZeyuChen) [inline examples/model_compression/pp-minilm/README.md:71]: 要补充上测试加速比的硬件信息，使用的精度情况，FP32还是FP16，加多一个NOTE

File: examples/model_compression/pp-minilm/README.md [hunk 2]
```diff
@@ -60,7 +60,15 @@ PP-MiniLM 压缩方案以面向预训练模型的任务无关知识蒸馏(Task-a
 
 2.量化后比量化前模型参数量多了 0.1M 是因为保存了 scale 值；
 
-3.性能测试的条件是：batch_size: 32, max_seq_len: 128。
+3.性能测试的条件是：
```
Comment (tianxin1860) [inline examples/model_compression/pp-minilm/README.md]: `性能测试的条件是:` -> `性能测试环境：` ？

**Review by tianxin1860 — COMMENTED** [review]
Leave a comment

File: examples/model_compression/pp-minilm/README.md (hunk 2 above)
Comment (LiuChiachi) [reply examples/model_compression/pp-minilm/README.md]: 感谢指出，已修改

File: examples/model_compression/pp-minilm/README.md (hunk 1 above)
Comment (LiuChiachi) [reply examples/model_compression/pp-minilm/README.md:71]: 感谢指出，已补充说明

**Review by ZeyuChen — APPROVED** [review]

**Pull request merged**